library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-    * Syncope <nowiki> {suggestedTag=Episode-phase, suggestedTag=Finding-significance-to-recording, suggestedTag=Episode-consciousness, suggestedTag=Episode-awareness, suggestedTag=Clinical-EEG-temporal-relationship, suggestedTag=Episode-event-count, suggestedTag=State-episode-start, suggestedTag=Episode-postictal-phase, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting} [Episode with loss of consciousness and muscle tone that is abrupt in onset, of short duration and followed by rapid recovery; it occurs in response to transient impairment of cerebral perfusion. Typical prodromal symptoms often herald onset of syncope and postictal symptoms are minimal. Syncopal convulsions resulting from cerebral anoxia are common but are not a form of epilepsy, nor are there any accompanying EEG ictal discharges.]</nowiki>
-    * Cataplexy <nowiki> {suggestedTag=Episode-phase, suggestedTag=Finding-significance-to-recording, suggestedTag=Episode-consciousness, suggestedTag=Episode-awareness, suggestedTag=Clinical-EEG-temporal-relationship, suggestedTag=Episode-event-count, suggestedTag=State-episode-start, suggestedTag=Episode-postictal-phase, suggestedTag=Episode-prodrome, suggestedTag=Episode-tongue-biting} [A sudden decrement in muscle tone and loss of deep tendon reflexes, leading to muscle weakness, paralysis, or postural collapse. Cataplexy usually is precipitated by an outburst of emotional expression-notably laughter, anger, or startle. It is one of the tetrad of symptoms of narcolepsy. During cataplexy, respiration and voluntary eye movements are not compromised. Consciousness is preserved.]</nowiki>
-    * Other-episode {requireChild}
+    * Background-activity-special-feature <nowiki> [Special Features. Special features contains scoring options for the background activity of critically ill patients. (Source: Beniczky ea 2017, Section 6.)]</nowiki>
+        ** Continuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Nearly-continuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Discontinuous-background-activity <nowiki> {suggestedTag=Rhythmic-activity-attribute, suggestedTag=Categorical-location-value, suggestedTag=Sensors} [Source: Beniczky ea 2017, Section 6.]</nowiki>
+        ** Background-burst-suppression <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors} [EEG pattern consisting of bursts (activity appearing and disappearing abruptly) interrupted by periods of low amplitude (below 20 microV) and which occurs simultaneously over all head regions. (Source: Beniczky ea 2013, Appendix S2; Beniczky ea 2017, Section 6.)]</nowiki>
```

Comment:
Suggest replacing _"(below 20 microV) and which occurs"_

with _"(below 20 microV) that occurs"_

Reply:
fixed